gateway/1.7/modules/ROOT/pages/flex-troubleshoot-linux-permissions.adoc

```diff
@@ -0,0 +1,53 @@
+= Troubleshooting Linux Permissions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+In Linux, to limit access to information about a Flex Gateway, the following `flexctl` commands require that user belong to the `flex` user group or to be the root user to execute them:
+
+* `flexctl dump`
+* `flexctl version`
+* `flexctl check connections`
+* `flexctl probe`
+* `flexctl registration inspect` (when ran without the `--file` flag)
+
+There are two options to execute these commands:
+
+* <<add-user-to-flex-group>>
+* <<run-as-root-user>>
+
+[[add-user-to-flex-group]]
+== Add the User to the Flex User Group
+
+Execute the following command to add the current user to the `flex` user group:
+
+[source,ssh]
+----
+sudo usermod -a -G flex $USER
+----
+
+After running the command, the user must log in to the VM again to begin using the `flexctl` commands as normal.
```

Comment:
Not sure about "as normal". Perhaps just leave it out? Or replace "to being using" ... "as normal" with "resume using"?

Reply:
Just left out